sdk/core/core-rest-pipeline/test/defaultRetryPolicy.spec.ts

```diff
@@ -36,26 +36,29 @@ describe("defaultRetryPolicy", function () {
   });
 
   it("It should give up after the default maxRetries is reached", async () => {
+    const errorLists = ["ETIMEDOUT", "ESOCKETTIMEDOUT", "ECONNREFUSED", "ENOENT", "ENOTFOUND"];
```

Comment:
nit: you could `["ETIMEDOUT", "ESOCKETTIMEDOUT", "ECONNREFUSED", "ENOENT", "ENOTFOUND"].forEach` around `it()` so it's one test per code.

Reply:
Updated in the new commit. Thank you for the suggestion 👍